guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -15,6 +15,7 @@ ifdef::foreman-el,foreman-deb,katello[]
 For more information, see {ReleaseNotesDocURL}[Release notes].
 endif::[]
 * {Project} {ProjectVersion} or {SmartProxy} {ProjectVersion} running on {EL} 8.
+Note that you can upgrade from any {EL} 8 minor version to {EL} 9 using LEAPP.
```

Comment:
This is not correct. https://access.redhat.com/articles/4263361 documents the supported upgrade paths for Leapp.
You need to be on RHEL 8.10 to go to 9.
(Not that running on non-latest RHEL 8 would be supported for Foreman/Satellite itself).